liquidprompt

```diff
@@ -1236,6 +1236,10 @@ _lp_kubernetes_context() {
     local kubernetes_context
     kubernetes_context=$(kubectl config current-context 2>/dev/null) || return 1
 
+    if [[ -n "$LP_DELIMITER_KUBECONTEXT_STRIP_PREFIX" ]]; then
+        kubernetes_context="${kubernetes_context#*${LP_DELIMITER_KUBECONTEXT_STRIP_PREFIX}}"
```

Comment:
One `#` will remove the shortest matching pattern, where the suffix check is greedy, removing the longest. Does it make sense to have them different?

Either way, both the prefix and suffix config options need clarifying documentation about their greediness, or lack thereof.

Reply:
I agree, it is a good use case. And if both prefix and suffix are supported then the variable names can be improved into a "symmetric" pair of variables.

Regarding the matching behavior: I would expect both variants to use greedy patterns in order to remove the longest part and keep the shortest path, i.e. one "element".

Using the EKS context `arn:aws:eks:$REGION:$ACCOUNT_ID:cluster/$CLUSTER_NAME` we do not see a difference if we split on "/", but it becomes clear if we split on ":". In my opinion the expected results are:
`LP_DELIMITER_KUBECONTEXT=":"` --> `arn`
`LP_DELIMITER_KUBECONTEXT_STRIP_PREFIX=":"` --> `cluster/$CLUSTER_NAME`

So this line 1240 should use the greedy expression 
`kubernetes_context="${kubernetes_context##*${LP_DELIMITER_KUBECONTEXT_STRIP_PREFIX}}"`

Reply:
Agree, greedy match is the most-likely desired case. Done and documented in 41d62571a3e84807ced4cf8143ff071bab907f5c